### ISSUE TITLE
Configure elastic search queries 

### ISSUE THREAD

**Issue by pirwlan** [DarioBalinzo/kafka-connect-elasticsearch-source#45]
Dear @DarioBalinzo,

I was having a look at your repo and find it really useful, thank you!
Playing around with it, I could not find any way to configure specific search query to limit the retrieved documents. Am I overlooking something, or is this not possible at the moment?

**Comment by DarioBalinzo** [comment]
Hi @pirwlan, I'm happy to hear that!

You can limit the maximum number of document retrieved by the connector using the `batch.max.rows` settings as described in the documentation.

This setting control the amount of data that is fetched from elastic in a single request.

**Comment by kranthicdac** [comment]
Can we pass the query as a parameter in the connector to filter the data from the Elasticsearch index?

**Comment by DarioBalinzo** [comment]
Yes, I will implement it in the next version of the connector among the other requests. I will probably ship the new version at the beginning of the 2022. If you need this feature to be prioritized please consider a small donation to the project.
